### ISSUE TITLE
[[inputs.ping]] when a URL does not exists

### ISSUE THREAD

**Issue by mertbakir** [influxdata/telegraf#6808]
Telegraf. 1.13.0

```conf
[[inputs.ping]]
  urls = ["server1", "server2", "server3", "server4", "server5"]
```

When server3 is not a resolved, ping will return "could not find host server3" and in this case inputs.ping doesn't look for server4 and server5. Telegraf keeps working and generating related values for server1 and server2.

I'm not sure if this is a bug but I expect it to log the error about server 3 and move on with server4.

**Comment by danielnelson** [comment]
This sounds a lot like https://github.com/influxdata/telegraf/issues/6690, but this should be fixed in 1.13.0.

I am testing with this configuration, and the non-localhost server is not up:
```
[[inputs.ping]]
   urls = ["localhost", "localhost", "debian-stretch-docker.virt", "localhost", "localhost", "localhost", "localhost"]
```

```
2019-12-17T05:19:35Z E! [inputs.ping] Error in plugin: lookup debian-stretch-docker.virt on 127.0.0.1:53: no such host
> ping,url=debian-stretch-docker.virt result_code=1i 1576559975000000000
> ping,url=localhost average_response_ms=0.028,maximum_response_ms=0.028,minimum_response_ms=0.028,packets_received=1i,packets_transmitted=1i,percent_packet_loss=0,result_code=0i,standard_deviation_ms=0,ttl=64i 1576559975000000000
> ping,url=localhost average_response_ms=0.022,maximum_response_ms=0.022,minimum_response_ms=0.022,packets_received=1i,packets_transmitted=1i,percent_packet_loss=0,result_code=0i,standard_deviation_ms=0,ttl=64i 1576559975000000000
> ping,url=localhost average_response_ms=0.021,maximum_response_ms=0.021,minimum_response_ms=0.021,packets_received=1i,packets_transmitted=1i,percent_packet_loss=0,result_code=0i,standard_deviation_ms=0,ttl=64i 1576559975000000000
> ping,url=localhost average_response_ms=0.019,maximum_response_ms=0.019,minimum_response_ms=0.019,packets_received=1i,packets_transmitted=1i,percent_packet_loss=0,result_code=0i,standard_deviation_ms=0,ttl=64i 1576559975000000000
> ping,url=localhost average_response_ms=0.02,maximum_response_ms=0.02,minimum_response_ms=0.02,packets_received=1i,packets_transmitted=1i,percent_packet_loss=0,result_code=0i,standard_deviation_ms=0,ttl=64i 1576559975000000000
> ping,url=localhost average_response_ms=0.016,maximum_response_ms=0.016,minimum_response_ms=0.016,packets_received=1i,packets_transmitted=1i,percent_packet_loss=0,result_code=0i,standard_deviation_ms=0,ttl=64i 1576559975000000000
```

**Comment by sspaink** [comment]
Closing because it seems this was resolved in an older version of Telegraf, please re-open if your still experiencing this issue.
